manager/main.go

```diff
@@ -135,6 +136,12 @@ func run(namespace string, metricsAddr string, enableLeaderElection bool,
 			}
 		}()
 
+		query, err := adminconfig.PrepareQuery()
```

Comment:
Add comment please describing that this is pre-compiling the config policies

Reply:
Done.